data/campaigns/fri02.wmf/scripting/init.lua

```diff
@@ -20,19 +20,46 @@ map = game.map
 p1_start = map.player_slots[1].starting_field
 difficulty = wl.Game().scenario_difficulty
 
-include "map:scripting/texts.lua"
-
 total_soldiers = {40, 25, 10}
 total_soldiers = total_soldiers[difficulty]
 takeover_soldiers = {15, 10, 5}
 takeover_soldiers = takeover_soldiers[difficulty]
+
 campaign_data = game:read_campaign_data("frisians", "fri01")
 if not campaign_data then
-   campaign_message_box(campaign_data_warning)
+   campaign_message_box({
+      title = _"Warning",
```

Comment:
```suggestion
      -- TRANSLATORS: Message title for a piece of information
      title = _"Note",
```

I think this would sound friendlier.